base_comment_template/__manifest__.py

```diff
@@ -1,14 +1,16 @@
 # -*- coding: utf-8 -*-
 # © 2013-2014 Nicolas Bessi (Camptocamp SA)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-{'name': 'Base Comments Templates',
- 'summary': 'Comments templates on documents',
- 'version': '9.0.1.0.0',
- 'license': 'AGPL-3',
- 'depends': ['base', ],
- 'author': 'Camptocamp,Odoo Community Association (OCA)',
- 'data': ['views/comment.xml',
+
+{"name": "Base Comments Templates",
```

Comment:
Please revert quoting type changes unless strictly required, to keep diff minimal. Or take the chance to fix this weird indentation and use the one widely used [as per the template](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py).